crates/generator/src/generator.rs

```diff
@@ -736,6 +736,33 @@ impl Generator {
         &self.backend_manage
     }
 
+    pub fn get_polyjuice_creator_id<S: State + CodeStore>(
```

Comment:
This function can be defined in `utils` module since it is not related to generator's logic, we can pass the backend manage as the parameter.